browserassets/js/browserOutput.js

```diff
@@ -628,6 +637,13 @@ $(function() {
         opts.highlightColor = savedConfig.shighlightColor;
         output('<span class="internal boldnshit">Loaded highlight color of: '+savedConfig.shighlightColor+'</span>');
     }
+    if (savedConfig.stheme) {
+        var body = $('body');
+        body.removeClass(opts.currentTheme);
+        body.addClass(savedConfig.stheme);
```

Comment:
Incredible nitpick, this might as well just be:
`$('body').removeClass(opts.currentTheme).addClass(savedConfig.stheme);`